### PR TITLE
Graphql tests

### PR DESCRIPTION
**Pull request by rotimi-babalola**
Tests for schema and graphql api

### CODE REVIEW

File: server/.babelrc [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "presets": ["env"],
+  "presets": ["env", "es2015"],
```
Comment (ghost) [inline server/.babelrc]: The `env` preset includes `es2015` and `es2017`. Not sure you still need the `es2015`.

File: server/tests/server/expense.server.test.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+import { dropDb } from '../helpers/testHelpers';
+import {
+  createNewExpenseMutation,
+  createNewEditExpenseMutation,
+  deleteExpenseMutation
+} from '../helpers/mutations';
+
+const server = supertest.agent(app);
+
+let expenseId;
```
Comment (ghost) [inline server/tests/server/expense.server.test.js:13]: Is this variable reassigned?

File: server/.babelrc (hunk 1 above)
Comment (rotimi-babalola) [reply server/.babelrc]: Cool

File: server/tests/server/expense.server.test.js (hunk 2 above)
Comment (rotimi-babalola) [reply server/tests/server/expense.server.test.js:13]: Yep

Comment (rotimi-babalola) [reply server/tests/server/expense.server.test.js:13]: On line 41

**Pull request merged**